sunlight-shadow/SunpathTool/csvanalyser.py

```diff
@@ -0,0 +1,28 @@
+import matplotlib.pyplot as plt
```

Comment:
What's the objective of this file ? Please add a small description in the beginning of each file.